src/pages/gen2/build-a-backend/data/customize-authz/signed-in-user-data-access/index.mdx

```diff
@@ -64,7 +64,7 @@ const schema = a.schema({
     .model({
       content: a.string(),
     })
-    .authorization([a.allow.private('identityPool')]),
+    .authorization(allow => [allow.authenticated('iam')]),
```

Comment:
Ditto, do we need to change line 58 around `identityPool`?

Reply:
this is a QA issue. Already filed a separate bug.